### PR TITLE
[MRG] Readme does not overwrite existing READMEs 

### PR DESCRIPTION
**Pull request by adam2392**
PR Description
--------------

Closes: #551 

Merge checklist
---------------

Maintainer, please confirm the following before merging:

- [ ] All comments resolved
- [ ] This is not your own PR
- [ ] All CIs are happy
- [ ] PR title starts with [MRG]
- [ ] [whats_new.rst](https://github.com/mne-tools/mne-bids/blob/master/doc/whats_new.rst) is updated
- [ ] PR description includes phrase "closes <#issue-number>"

### CODE REVIEW

File: mne_bids/write.py [hunk 1]
```diff
@@ -1042,8 +1042,9 @@ def write_raw_bids(raw, bids_path, events_data=None,
     unit = UNITS.get(ext, 'n/a')
     manufacturer = MANUFACTURERS.get(ext, 'n/a')
 
+    # XXX: can include README overwrite in future if using a template API
     # save all participants meta data and readme file
```
Comment (sappelhoff) [inline mne_bids/write.py:1046]:
```suggestion
    # save readme file unless it already exists
    # XXX: can include README overwrite in future if using a template API
    # XXX: see https://github.com/mne-tools/mne-bids/issues/551
```

**Review by sappelhoff — COMMENTED** [review]
fine with me :+1:

**Comment by agramfort** [comment]
I think your problem is here:

https://github.com/mne-tools/mne-bids/blob/f64721cf11912ff5fdc56bf2ab481ab734bf3682/mne_bids/write.py#L232

you pass overwrite=True and not overwrite=overwrite

>

**Comment by adam2392** [comment]
> I think your problem is here: https://github.com/mne-tools/mne-bids/blob/f64721cf11912ff5fdc56bf2ab481ab734bf3682/mne_bids/write.py#L232 you pass overwrite=True and not overwrite=overwrite
> […](#)

This seems fair since it adds it onto the end of the README?

**Comment by agramfort** [comment]
thx @adam2392

**Pull request merged**